packages/utils-compat-data/README.md

```diff
@@ -0,0 +1,172 @@
+ids to check. This package accepts two different formats of browsers
+ids: browserslist or MDN.
+
+The friendly names and ids for each one are as follows:
+
+| Browser name | [Browserslist][bl browsers] | [MDN][mdn browsers] |
+| ------------ | ------------ | --------- |
+| Chrome       | chrome       | chrome |
+| Chrome Android | and_chr    | chrome_android |
+| Edge         | edge         | edge |
+| Firefox      | ff           | firefox |
+| Firefox Android | and_ff    | firefox_android |
+| Internet Explorer | ie      | ie |
+| Opera        | opera        | opera |
+| Opera Android | op_mobile   | opera_android |
+| QQ Android   | and_qq       | qq_android |
+| Safari       | safari       | safari |
+| Safari iOS   | ios_saf      | safari_ios |
+| Samsung Internet | samsung  | samgsunginternet_android |
+| UC Android   | and_uc       | uc_android |
+| Webview Android | android   | webview_android |
+
+You can check the next sections for example on how to use `FeatureQuery`s
+and list of browsers.
+
+### `getFriendlyName`
+
+Get the friendly name of a browser from an id.
+
+```js
+import { getFriendlyName } from '@hint/utils-compat-data';
+
+console.log(getFriendlyName('and_ff')); // "Firefox Android"
+console.log(getFriendlyName('ie')); // "Internet Explorer"
+```
+
+### `getUnsupported`
+
+Get browsers without support for CSS or HTML features.
+
+It accepts a `FeatureQuery` and a list of browsers.
+
+```js
+import { getUnsupported } from '@hint/utils-compat-data';
+
+console.log(getUnsupported({ element: 'details' }, ['chrome 74', 'ie 11'])); // ['ie 11']
+console.log(getUnsupported(
+    {
+        attribute: 'rel',
+        element: 'link',
+        value: 'noopener'
+    },
+    ['edge 12', 'firefox 63'])); // ['edge 12']
+```
+
+### `getUnsupportedDetails`
+
+Get browsers without support with details on when support was added or removed.
+
+Similar to `getUnsupported`, but returns an object with both a list of:
+
+* `browsers` which were unsupported and a map of browsers to `browserDetails`
+* to get additional information (e.g. what version the feature is added in).
+
+It accepts a `FeatureQuery` and a list of browsers.
+
+```js
+import { getUnsupportedDetails } from '@hint/utils-compat-data';
+
+const unsupportedInfo = getUnsupportedDetails(
+    { property: 'appearance' },
+    ['chrome 75', 'firefox 63']);
+```
+
+`unsupportedInfo` will look something like the following:
+
+```json
+{
+    "browsers": [ "chrome 75", "firefox 63" ],
+    "details": {
+        "chrome 75": {
+            "alternative": {
+                "name": "-webkit-appearance",
+                "versionAdded": 1
+            }
+        },
+        "firefox 63": {
+            "alternative": {
+                "name": "-moz-appearance",
+                "versionAdded": 1
+            }
+        }
+    }
+}
+```
+
+**Note**: `details` is a `Map` that uses the browser versions as keys.
+
+### `isSupported`
+
+Query MDN for support of CSS or HTML features.
```

Comment:
We should call out that this requires *all* provided browsers to have support in order to return `true`. If any single provided browser is not supported this will return `false`.